### DISCUSSION TITLE
Welcome to LexiLaw Discussions!

### DISCUSSION THREAD

**Discussion by CSHaitao** [CSHaitao/LexiLaw#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hubeiler** [comment]
python=3.9
transformers==4.28.1
tqdm==4.64.1
datasets==2.8.0
pytorch==1.12.1
deepspeed==0.9.1
peft==0.3.0 /when Python 3.12.5,how others match it ?

**Comment by hubeiler** [comment]
怎么没有更新，能不能在Ollama上运行呢？
